platform/security/common/src/main/java/ddf/security/ws/policy/impl/PolicyWSDLGetInterceptor.java

```diff
@@ -84,7 +84,7 @@ public void handleMessage(Message message) throws Fault {
       Document doc = getDocument(message, baseUri, map, ctx);
 
       // DDF- start ADDED code
-      if (map.containsKey("wsdl")) {
+      if (map.containsKey("wsdl") && doc != null) {
```

Comment:
✏️  reverse the 2 conditions such that we don't bother hashing the map if doc was null to start with